### DISCUSSION TITLE
Storing application state (key => value)

### DISCUSSION THREAD

**Discussion by Feofilov** [isar/isar#850] [Help]
Is it advisable to considered Isar as a application state (key => value) storage. Such as theme settings (dark/light), authorization tokens, etc. Or is Isar only for storing collections and for such states it is better to use shared_preferences? I really like Isar because I can store any type of data. And I do not want to have two databases in the project (unless, of course, shared_preferences can be called a database).

**Comment by MrCsabaToth** [comment]
The https://github.com/isar/isar/discussions/1066 is same as this
